draft-timbru-sidrops-publication-server-bcp-02.md

```diff
@@ -120,13 +120,42 @@ needed during these windows.
 
 In this section, we will elaborate on the following recommendations:
 
+  - Bandwidth
   - Use a separate hostname: do not share fate with rsync or the Publication Server.
   - Use a CDN if possible
   - Use randomized filenames for Snapshot and Delta Files
   - Limit the size of the Notification File
   - Combine deltas to limit the size of the Notification File
   - Timing of publication of Notification File
 
+## RRDP bandwidth
+
+The bandwidth needed for RRDP evolves and depends on many parameters. These
+consist of three main groups:
+
+  1. RRDP-specific repository properties, such as the size of notification-,
+     delta-, and snapshot files.
+  2. Properties of the CAs publishing in a repository, such as the number of
+     updates, number of objects, and size of objects.
```

Comment:
Perhaps we should measure, but I suspect that MFT/CRL renewal is one of the biggest factors.